app/kubernetes-ingress-controller/2.0.x/references/annotations.md

```diff
@@ -72,13 +65,22 @@ KongConsumer resources by default.
 
 ### kubernetes.io/ingress.class
 
+<div class="alert alert-ee blue">
+<a href="https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation">Kubernetes versions after 1.18</a>
+introduced the new <code>ingressClassName</code> field to the Ingress spec
+and deprecated the <code>kubernetes.io/ingress.class</code> annotation.
+Ingress resources should now use the <code>ingressClassName</code> field.
+Kong resources (KongConsumer, TCPIngress, etc.) still use the
+<code>kubernetes.io/ingress.class</code> annotation.
+</div>
```

Comment:
```suggestion
{:.note}
> Kubernetes versions after 1.18 introduced the new `ingressClassName` 
field to the Ingress spec and 
[deprecated the `kubernetes.io/ingress.class` annotation](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation). 
Ingress resources should now use the `ingressClassName` field. 
Kong resources (KongConsumer, TCPIngress, etc.) 
still use the `kubernetes.io/ingress.class` annotation.
```
You don't need to use raw HTML anymore for alerts, check out the style guide doc we posted [here](https://docs.konghq.com/contributing/markdown-rules/#admonitions). The old raw HTML format _will_ work, but ideally, we should avoid it going forward.

Also moved the URL to link from the deprecated property, otherwise the context was a bit confusing.